kilkari/src/test/java/org/motechproject/nms/kilkari/osgi/SubscriptionServiceBundleIT.java

```diff
@@ -300,6 +300,29 @@ public void testCreateSubscriptionExistingSubscriberDifferentLanguage() throws E
     }
 
     @Test
+    public void testCreateSubscriptionExistingSubscriberWithoutLanguage() throws Exception {
```

Comment:
is there a test for the negation of this one?...where the subscriber has a location  and so does the subscription, to make sure that it doesn't get overwritten?